.github/CODEOWNERS

```diff
@@ -311,7 +311,7 @@
 /sdk/translation/                                                    @kristapratico @mohamedshabanofficial
```

Comment:
FYI @kristapratico  mohamedshabanofficial exists here as well.

Reply:
Fixed it here: https://github.com/Azure/azure-sdk-for-python/pull/30918